File: practice/friendly pair.py
num1= 6
num2= 28

def Factors(num):
    sum = 0
    for i in range(1,num):
        if(num%i==0):
            sum+=i
    return sum/num
    
if(Factors(num1)== Factors(num2)):
    print(f"{num1} and {num2} are Friendly pairs")
else:
    print(f"{num1} and {num2} are not friendly pairs")

File: c-language/FriendlyPair.c
#include <stdio.h>
int factor(int num){
    int sum = 0;
    for(int i = 1; i<num;i++){
        if(num%i==0)
        sum=sum+i;
    }
    return sum;
}
int main(){
    int num1 = 6;
    int num2= 28;
    int sum1= factor(num1);
    int sum2=factor(num2);
    if(sum1/num1== sum2/num2)
        printf("Friendly pair");
    else
        printf("Not Friendly pair");


}